python/pyarrow/parquet/core.py

```diff
@@ -1108,6 +1108,18 @@ def close(self):
         if self.file_handle is not None:
             self.file_handle.close()
 
+    def add_key_value_metadata(self, key_value_metadata):
+        """
+        Add key-value metadata to the file.
+        This will overwrite any existing metadata with the same key.
+
+        Parameters
+        ----------
+        key_value_metadata : {Key, Value}
```

Comment:
Let's reuse the same conventions as in other docstrings:
```suggestion
        key_value_metadata : dict
            Keys and values must be string-like / coercible to bytes
```